roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -32,7 +32,6 @@
         ) |
         combine(
           {
-            'DATAPLANE_REPO': operators_build_output['dataplane-operator'].git_src_dir,
```

Comment:
This dict from line 33 to 39, needs to be removed.

Reply:
+1